internal/internal_test.go

```diff
@@ -1062,7 +1062,7 @@ func TestGetResponseHttpStatus(t *testing.T) {
 		t.Fatal("Expected error but got nil")
 	}
 
-	input["http_status"] = json.Number(301)
+	input["http_status"] = json.Number("1")
```

Comment:
I suspect that this can disappear when rebasing this branch. It had been dealt with in #201.